CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 - [#227](https://github.com/pusher/oauth2_proxy/pull/227) Add Keycloak provider (@Ofinka)
 - [#273](https://github.com/pusher/oauth2_proxy/pull/273) Support Go 1.13 (@dio)
 - [#275](https://github.com/pusher/oauth2_proxy/pull/275) docker: build from debian buster (@syscll)
+- [#258](https://github.com/pusher/oauth2_proxy/pull/258) Add IDToken for Azure provider
+  - This PR adds the IDToken into the session for the Azure provider allowing requests to a backend to be identified as a specific user. As a consequence, if you are using a cookie to store the session the cookie will now exceed the 4kb size limit and be split into multiple cookies. This can cause problems when using nginx as a proxy, resulting in no cookie being passed at all. Either increase the proxy_buffer_size in nginx or implement the redis session storage (see https://pusher.github.io/oauth2_proxy/configuration#redis-storage)
```

Comment:
I feel like this note is definitely relevant but maybe the changelog is the wrong place? WDYT @JoelSpeed @steakunderscore?

Reply:
I am happy to have a note like this in the changelog, it's important that people are aware of changes like this that may break their setups. 

It's the sort of thing I'll copy out to highlight it in the release highlights when I cut the next release

Reply:
Totally happy for it to stay in the changelog :)